src/states/game.js

```diff
@@ -115,12 +127,28 @@ Game.prototype =
         {
             this.debug_graphics = this.game.add.graphics();
         }
+        
+        for (size = 1; size < 4; size++) { 
```

Comment:
Here, size is created in global scope. if you type window.size into the console you can see it. Not a massive problem but we can easily avoid this with ```for (var size = 1...```